File: Logistics.Data/InventoryItem.cs
namespace Logistics.Data
{
    public class InventoryItem
    {
        public int Id { get; set; }
        public int InventoryId { get; set; }
        public int ItemId { get; set; }
        public bool IsAccounted { get; set; }
        public bool IsMissing { get; set; }
        public string Remarks { get; set; }

        public Inventory Inventory { get; set; }
        public Item Item { get; set; }
    }
}

File: Logistics.Web/ClientApp/src/app/models/inventory-item.ts
import { Inventory } from './inventory';
import { Item } from './item';

export class InventoryItem {
  id: number;
  inventoryId: number;
  itemId: number;
  isAccounted: boolean;
  isMissing: boolean;
  remarks: string;
  inventory: Inventory;
  item: Item;
}
